### PR TITLE
Update aws sdk

### PR DESCRIPTION
**Pull request by zvymazal**

### CODE REVIEW

File: Gopkg.lock [hunk 1]
```diff
@@ -65,70 +84,164 @@
...
+    "github.com/aws/aws-sdk-go/aws/session",
+    "github.com/aws/aws-sdk-go/service/dynamodb",
+    "github.com/aws/aws-sdk-go/service/dynamodb/dynamodbattribute",
+    "github.com/aws/aws-sdk-go/service/dynamodb/dynamodbiface",
+    "github.com/aws/aws-sdk-go/service/kms",
+    "github.com/aws/aws-sdk-go/service/kms/kmsiface",
+    "github.com/olekukonko/tablewriter",
+    "github.com/stretchr/testify/assert",
+    "github.com/stretchr/testify/mock",
+    "github.com/versent/unicreds",
```
Comment (TheSAS) [inline Gopkg.lock]: Vivino?

**Review by TheSAS — APPROVED** [review]
LGTM.
How did you generated all those mocks and locks?

**Comment by zvymazal** [comment]
```
dep ensure -update
mockery --dir ../../aws/aws-sdk-go/service/dynamodb/dynamodbiface --all
mockery --dir ../../aws/aws-sdk-go/service/kms/kmsiface --all
```

**Comment by zvymazal** [comment]
@TheSAS Updated dependencies for generating mocks. We should be independent from the upstream repo now.

**Pull request merged**